source/mainnet/net/snippets/delegation-faq.rst

```diff
@@ -0,0 +1,98 @@
+
+**How do I activate a baker pool?**
+All new bakers come with pool support built in. You can actively choose to disable this feature upon creating your baker keys. Read more about setting up a baker here: alsdkflsadkjfsdjfk
+
+**How often are rewards distributed to bakers?**
+If your baker is chosen to bake the next block on the blockchain you will receive rewards for this. All rewards are distributed at :ref:`pay day<glossary-pay-day>` which is once every 24 hours. If your baker is also baking CCDs delegated to it, the pool owner rewards from the delegated stake (if any) are also distributed each pay day.
+
+**What does it mean for a baker to accept delegators?**
+To open a baker pool for delegators means that users who want to earn rewards but perhaps don't want to be a baker, can delegate some of their stake to a baker pool. By doing this, they give the baker the right to bake with their stake. When they delegate their stake to the baker this increases the baker’s stake and thus the odds of being selected to bake a block. This increase in rewards is then shared with the delegator.
+
+**Are there any limits to the size of a baker pool?**
+Concordium blockchain imposes two caps on the amount of stake in a single pool:
+
+A pool cannot have more than 10% of all staked CCD
+The total stake of a pool cannot be more than 3x the size of the pool owners own stake
+
+The first limit ensures that the blockchain remains distributed and prevents the crash of one baker from affecting the whole system. A party with lots of capital is welcome to run multiple bakers to avoid this cap as in turn this would increase decentralization.
+
+The second cap is for security reasons as it prevents a party with a small stake from controlling too large a part of the system using leverage.
+
+**What happens when the maximum stake cap is reached for my baker?**
+These two caps are soft caps in the sense that a pool can have more stake than allowed, but only the allowed amounts are taken into account in the consensus algorithm and when distributing rewards. A pool violating the amount limits will gradually receive less rewards per CCD in the pool as the size of the pool continues to increase.
+
+**How much less/more can I expect to earn when opening my pool to delegators?**
+The amount you earn depends on how much stake your pool is baking. The more delegators, the more stake you have and the greater your chances of being selected to bake a block are. 10% of the additional rewards the delegators’ stake in your pool attracts will go to the pool owner. As you are able to have 2x your own stake delegated to your pool, you are effectively able to increase your rewards by 20% if your pool is full.
+
+Therefore, attracting delegators and managing your baker responsibly will maximize rewards for all. In :ref:`Baker management<baker-pool>` you'll find information about how to be a responsible baker.
+
+**Is there a cool-down period for bakers?**
```

Comment:
If you are new to the game, would you know what a cool-down is and what triggers it?